src/utils/ValidComponentChildren.js

```diff
@@ -137,11 +135,26 @@ function some(children, func, context) {
   return result;
 }
 
+function toArray(children) {
```

Comment:
Not really – it's just `map` with the identity function as the predicate: https://github.com/facebook/react/blob/24dfb56113a214d98f29f69e2c26f67d7e947067/src/isomorphic/children/ReactChildren.js#L196-L205